compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -17,6 +17,9 @@ class ScalaSettings extends Settings.SettingGroup {
   val javabootclasspath: Setting[String] = PathSetting("-javabootclasspath", "Override java boot classpath.", Defaults.javaBootClassPath) withAbbreviation "--java-boot-class-path"
   val javaextdirs: Setting[String] = PathSetting("-javaextdirs", "Override java extdirs classpath.", Defaults.javaExtDirs) withAbbreviation "--java-extension-directories"
   val sourcepath: Setting[String] = PathSetting("-sourcepath", "Specify location(s) of source files.", Defaults.scalaSourcePath) withAbbreviation "--source-path"
+  val scansource: Setting[Boolean] = BooleanSetting("-scansource", "Scan source files to locate classes for which class-name != file-name") withAbbreviation "--scan-source"
+  val sourceroot: Setting[String] = PathSetting("-sourceroot", "Specify workspace root directory", ".")
+  val targetroot: Setting[String] = PathSetting("-targetroot", "Specify output directory", "")
```

Comment:
Why have both -targetroot and -d? Should we not just have one?

Reply:
it was being used in the extractSemanticDB phase to provide a different destination for the semanticdb files than the class files

Reply:
but defaulting to `-d` if not provided

Reply:
so if we keep it, the setting should be renamed, or at least description adjusted